config/CMakeLists.txt

```diff
@@ -22,3 +22,4 @@
 #    CMake for hpcc nagios and nrpe config
 #####################################################
 INSTALL(FILES hpcc_nrpe.cfg DESTINATION   /etc/nagios/nrpe.d/)
+INSTALL(FILES hpcc_nagios.cfg DESTINATION /etc/nagios3/conf.d/)
```

Comment:
No, hpcc_nagios.cfg is part of our hpcc-nagios package.  It describes the the health check utilities included in our package and used by nagios for hpcc specific health checks.  It should be installed as part of hpcc-nagios package.